### PR TITLE
Initial testing version of a complex Multimedia atom

### PR DESCRIPTION
**Pull request by fredex42**
as we discussed :)

### CODE REVIEW

**Comment by mchv** [comment]
@fredex42 could you rebase your pull-request from master? thanks

File: src/main/thrift/atoms/multimedia.thrift [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  // content-atom wrapping?
+  // should we store an embed count here or is this done higher up?
+  1  : required string id
+  2  : required string title
+  3  : required i16 contentVersion  //from __version field in Vidispine
+  7  : required PublicationStatus publicationStatus
+  8  : required MultimediaSubtype mediaType
+  9  : required LegalStatus legalStatus
+  10 : required list<Chapter> chapters
+  //11 : required shared.User creator        //we might not be able to get these easily but it would be good to get them. Optional at top level
```
Comment (mchv) [inline src/main/thrift/atoms/multimedia.thrift]: I am not sure we can add later the `11` item without breaking compatibility

**Comment by philmcmahon** [comment]
Is this now obsolete? Can we close it?

**Comment by jamesgorrie** [comment]
@philmcmahon please do.

**Pull request closed without merging**